### DISCUSSION TITLE
Get your vote in!!! UI design

### DISCUSSION THREAD

**Discussion by inorrr** [CSC207-2022F-UofT/course-project-macrosoft#96] [Polls]
How does everyone think about this? 
<img width="1012" alt="Screen Shot 2022-12-01 at 2 54 24 PM" src="https://user-images.githubusercontent.com/94703030/205396063-df85f6eb-3c01-4b59-8221-b86499a5b977.png">
<img width="1012" alt="Screen Shot 2022-12-02 at 4 49 51 PM" src="https://user-images.githubusercontent.com/94703030/205396071-5147432e-c0cd-4a56-ac74-242cd781fb6b.png">

**Comment by ghost** [comment]
Nice and elegant design!

**Reply by jacksonsujh** [reply]
👍
